src/components/WorldMap.js

```diff
@@ -153,12 +161,25 @@ export class WorldMap extends Component {
             localStorage.setItem('geolocation', 'true');
             const { latitude, longitude } = location.coords;
             this.state.map.setView([latitude, longitude]);
+            locationSet='true';
           });
         } else {
           localStorage.removeItem('geolocation');
         }
       });
     }
+
+    if(locationSet='false'){
```

Comment:
Using a single = will do an assignment instead of comparison. A simple `if (!locationSet)` should be sufficient

Reply:
okay.. I think it got erased with find and replace...will correct it